docs/data/material/varnish/logos/logos.md

```diff
@@ -0,0 +1,54 @@
+---
+  title: Varnish AI2 Logos
+  githubLabel: 'varnish: logos'
+---
+
+# AI2 Logos
+
+<p class="description">AI2 Specific Logos as inline SVG elements.</p>
+
+## When To Use
+
+Whenever you need the official AI2 or project logo.
+
+## Examples
+
+### Basic Usage
+
+{{"demo": "LogosDefaultDemo.js"}}
+
+### Different Sizes
+#### You can change the size of the logo to be one of three values: _micro_, _default_ or _large_.
+
+{{"demo": "LogosSizesDemo.js"}}
+
+### Altered Text
+#### Depending on the context, you may or may not want to display the full organization name text.
+
+{{"demo": "LogosTextDemo.js"}}
+
+### White Logo
+#### If you're using the logo against a dark background, you will want to set the component's `color` value to `white`.
+
+{{"demo": "LogosWhiteDemo.js"}}
+
+### Canonical logos for AI2's projects.
+
+{{"demo": "ProjectsLogosDemo.js"}}
+
+## API
+
+### AI2 Logo
+
+| Property    | Description                | Type                         | Default   |
+| ----------- | -------------------------- | ---------------------------- | --------- |
+| size        | sets size of logo          | 'micro' \| 'default' \| 'lg' | 'default' |
```

Comment:
can you please convert to use 
xs, sm, md, lg, and xl
to be more like material?

Reply:
sure! i think sm, md, lg seems like the right ones to use for the 3 logo sizes?

Reply:
yep.
we should proactively try to align with material whenever possible